website/backend/settings.py

```diff
@@ -26,7 +27,7 @@
 SECRET_KEY = config('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = config('DEBUG', 'True') == 'True'
```

Comment:
I think this would be the same without `== 'True'` since we already set a default value for 'DEBUG'

Reply:
@123MwanjeMike could you help me share the file where the default value for DEBUG is set.

Reply:
Oh, it's python syntax. `config('DEBUG', 'True')` default value is True if no value is present for DEBUG.

Reply:
I think 'True' (a string) is not equal to True (a boolean).

Reply:
will the app be able to interpret `'True'` as `True` ? @123MwanjeMike

Reply:
Not really, the two would be different. Unless if it was set like `config('DEBUG', True)`.
I now get the idea behind the comparison check.

Reply:
@NoahNsimbe simple set the default to false 
```suggestion
DEBUG = config('DEBUG', False)
```
Incase some one wants it to be True, they can set DEBUG=True in their `.env` file. If you want it false,  you simple dont set it.

Reply:
or you explore using 'distutils`
```suggestion
from distutils.util import strtobool
DEBUG = strtobool(config('DEBUG', False))
```
With this, you can use string bools such as False in you `.env` file.